mkdocs.yml

```diff
@@ -173,6 +173,7 @@ nav:
     - Cron: ./reference/cron.md
     - Private Registries: ./reference/private-registries.md
     - TLS:
+      - Custom CA bundle: ./reference/ssl/custom-cert-bundle.md
```

Comment:
This shouldn't be the first item, but the second.

Reply:
I switched the order.

At the moment this setting is indeed only used for IAM so it could be moved there.

Although I think it might be used for other components as well in the future if a customer needs it. (I'm thinking of e.g. the queue-worker to connect to an external NATS server).